scripts/gen-tag-table.py

```diff
@@ -882,8 +882,8 @@ def rank_delta(bcp_47, ot):
 
 def hb_tag(tag):
     if tag == DEFAULT_LANGUAGE_SYSTEM:
-        return 'Tag(0)\t           '
-    return 'Tag::from_bytes(b\"%s%s%s%s\")' % tuple(('%-4s' % tag)[:4])
+        return 'Tag::new(&[0; 4])\t           '
```

Comment:
```suggestion
        return 'Tag::new(&[0; 4])'
```
The two branches produce strings of the same length, the manual left padding is not needed now.

Reply:
Fixed, ty!